pkg/scheduler/framework/interface.go

```diff
@@ -108,7 +108,7 @@ const (
 )
 
 // PodsToActivateKey is a reserved state key for stashing pods.
-// If the stashed pods are present in unschedulableQ or backoffQ，they will be
+// If the stashed pods are present in unschedulablePods or backoffQ，they will be
```

Comment:
I guess backoffQ is not a queue either :)

But we can fix that in a follow up

Reply:
```go
	// activeQ is heap structure that scheduler actively looks at to find pods to
	// schedule. Head of heap is the highest priority pod.
	activeQ *heap.Heap
	// podBackoffQ is a heap ordered by backoff expiry. Pods which have completed backoff
	// are popped from this heap before the scheduler looks at activeQ
	podBackoffQ *heap.Heap
```

BackoffQ is also a heap like activeQ. Maybe it can also be called queue?

Reply:
ah true... Nvm then.